algo-core/ortools_solver.py

```diff
@@ -970,12 +931,12 @@ def constraint_various_custom_conditions():
     for agent in agents_with_max_shift_hours:
         handle = agent["handle"]
         hours = int(agent["value"])
-        for t in range(num_tracks):
-            for d in range(num_days):
-                if not (handle in unavailable_agents[d]):
-                    model.Add(v_tdh.loc[(t, d, handle), "shift_duration"] <= hours)
+        for t, track in enumerate(tracks):        
```

Comment:
There are leftover empty spaces on this line and the ones below.

Reply:
> Your sentence reads like this would be a configuration option.

You can configure it in the download-and-configure-input.js file:
`const SCHEDULE_OPTS = {
...
	supportEndHour: 26, 
...
}

you can move it to 27 or back to 24

Reply:
thanks!